interceptors/protovalidate/example_stream_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+package protovalidate_test
+
+import (
+	"github.com/bufbuild/protovalidate-go"
+	protovalidate_middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/protovalidate"
```

Comment:
I don't know that we need to alias this, do we?

Reply:
Idk why, but `protovalidate-go` package defined as `protovalidate`, so I've set import alias.

I'd rather would suggest to rename my package to something like `bufvalidate`, what do you think about these?

Reply:
I'm confused, can't we just use it as `protovalidate`?

Reply:
Oh sorry, I see what you mean now. This is fine.